Assets/Tests/InputSystem/CoreTests_Analytics.cs

```diff
@@ -520,53 +520,56 @@ public void Analytics__ShouldReportBuildAnalytics_WheHavingSettingsAssetWithCust
-            Assert.That(data.featureParanoidReadValueCachingChecksEnabled, Is.True);
-            Assert.That(data.featureUseWindowsGamingInputBackend, Is.True);
-            Assert.That(data.featureDisableUnityRemoteSupport, Is.True);
-            Assert.That(data.featureRunPlayerUpdatesInEditMode, Is.True);
+            Assert.That(data.has_settings_asset, Is.False); // Note: We just don't write any file in this test, hence false
+            Assert.That(data.has_default_settings, Is.False);
+
+            Assert.That(data.update_mode, Is.EqualTo(InputBuildAnalytic.InputBuildAnalyticData.UpdateMode.ProcessEventsInFixedUpdate));
+            Assert.That(data.compensate_for_screen_orientation, Is.EqualTo(true));
+            Assert.That(data.default_deadzone_min, Is.EqualTo(0.4f));
+            Assert.That(data.default_deadzone_max, Is.EqualTo(0.6f));
+            Assert.That(data.default_button_press_point, Is.EqualTo(0.1f));
+            Assert.That(data.button_release_threshold, Is.EqualTo(0.7f));
+            Assert.That(data.default_tap_time, Is.EqualTo(1.3f));
+            Assert.That(data.default_slow_tap_time, Is.EqualTo(2.3f));
+            Assert.That(data.default_hold_time, Is.EqualTo(3.3f));
+            Assert.That(data.tap_radius, Is.EqualTo(customSettings.tapRadius));
+            Assert.That(data.multi_tap_delay_time, Is.EqualTo(customSettings.multiTapDelayTime));
+            Assert.That(data.background_behavior, Is.EqualTo(InputBuildAnalytic.InputBuildAnalyticData.BackgroundBehavior.IgnoreFocus));
+            Assert.That(data.editor_input_behavior_in_playmode, Is.EqualTo(InputBuildAnalytic.InputBuildAnalyticData.EditorInputBehaviorInPlayMode.AllDeviceInputAlwaysGoesToGameView));
+            Assert.That(data.input_action_property_drawer_mode, Is.EqualTo(InputBuildAnalytic.InputBuildAnalyticData.InputActionPropertyDrawerMode.MultilineEffective));
+            Assert.That(data.max_event_bytes_per_update, Is.EqualTo(customSettings.maxEventBytesPerUpdate));
+            Assert.That(data.max_queued_events_per_update, Is.EqualTo(customSettings.maxQueuedEventsPerUpdate));
+            Assert.That(data.supported_devices, Is.EqualTo(customSettings.supportedDevices));
+            Assert.That(data.disable_redundant_events_merging, Is.EqualTo(customSettings.disableRedundantEventsMerging));
+            Assert.That(data.shortcut_keys_consume_input, Is.EqualTo(customSettings.shortcutKeysConsumeInput));
+
+            Assert.That(data.feature_optimized_controls_enabled, Is.True);
+            Assert.That(data.feature_read_value_caching_enabled, Is.True);
+            Assert.That(data.feature_paranoid_read_value_caching_checks_enabled, Is.True);
+            Assert.That(data.feature_disable_unity_remote_support, Is.True);
+            Assert.That(data.feature_run_player_updates_in_editmode, Is.True);
+#if UNITY_INPUT_SYSTEM_PROJECT_WIDE_ACTIONS
             Assert.That(data.featureUseIMGUIEditorForAssets, Is.True);
```

Comment:
Looks like this line needs fixing

Reply:
Could elaborate on what you mean? I do not see it

Reply:
AH, sorry, will fix, Resharper failed to refactor

Reply:
Believe its fixed now

Reply:
OK to resolve?